tastytaps/taps/tests/test_serializers.py

```diff
@@ -0,0 +1,64 @@
+from django.test import TestCase
+
+from tastytaps.taps.models import Brewery, Price
+from tastytaps.taps.serializers import (BrewerySerializer, PriceSerializer,
+                                        TapsSerializer)
+
+
+class TestBaseSerializer(TestCase):
+    def setUp(self):
+        self.serializer = None
+        self.data = {}
+
+    def success_serialize(self):
+        self.assertTrue(self.serializer(data=self.data).is_valid())
+
+    def error_serialize(self):
+        self.assertFalse(self.serializer(data=self.data).is_valid())
+
+
+class TestBrewerySerializer(TestBaseSerializer):
+    def setUp(self):
+        super(TestBrewerySerializer, self).setUp()
+        self.serializer = BrewerySerializer
+        self.data = {'name': 'Oakshire Brewing', 'city': 'Eugene'}
+
+    def test_success_serialize(self):
+        return self.success_serialize()
+
+    def test_error_serialize(self):
+        del self.data['name']
+        return self.error_serialize()
```

Comment:
I think it would be good to also test the deserialization. E.g. by calling `self.serializer(obj).data` then asserting that `data['name']` matches the `obj`, etc. Right now the serializers are simple but in the future if we customize them it might be nice to have the tests there. What do you think?